docs/howtos/namespaces.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_label: "Working With Multiple Namespaces"
+sidebar_position: 3
+title: ""
+---
+
+# How to work with multiple namespaces
+
+Like Kubernetes, Epinio has the concept of namespaces, and means of working with them.
+
+Working with multiple of them however is different to Kubernetes.
+
+In Kubernetes the `kubectl` client accepts an option `--namespace` for all its commands,
+which provide the name of the namespace to work with.
+
+Epinio on the other hand maintains a __current namespace__ in its local state, often
+also called the __targeted namespace__.
+
```

Comment:
For context to the user, perhaps it may be worth to mention:
- That Epinio brings namespace `workspace` as default.
- That prior targeting namespace `foo` and `bar` it is actually needed to have them created first.

Reply:
Resolved with commit d0e745295ac49d97abc5d5b55d2c1495d7efb574